### PR TITLE
fix: disabled on-chain hash update, added countdown until user can migrate

### PR DESCRIPTION
**Pull request by truemiller**
includes @jmoreira-valory's [fix: disable hash update](https://github.com/valory-xyz/olas-operate-app/commit/7d1c6ef86d2913b5651fd15fc7da5a9fe308fa43)

added [fix: migration countdown, and minimum duration fix from staging](https://github.com/valory-xyz/olas-operate-app/commit/5d2720db93adcb9158924ea7f838134c4807f27e)

copied fix for hiding funding options (merged earlier to staging) when user is not able to migrate due to staking duration

### CODE REVIEW

File: operate/services/manage.py [hunk 1]
```diff
@@ -559,8 +559,9 @@ def _deploy_service_onchain_from_safe(  # pylint: disable=too-many-statements,to
             (not is_first_mint)
             and (on_chain_hash is not None)
             and (
-                on_chain_hash != service.hash
-                or current_agent_id != staking_params["agent_ids"][0]
+                # TODO Discuss how to manage on-chain hash updates with staking programs.
+                # on_chain_hash != service.hash or  # noqa
```
Comment (0xArdi) [inline operate/services/manage.py:563]: emphasizing that i agree with this change

**Review by 0xArdi — APPROVED** [review]

File: frontend/components/ManageStakingPage/StakingContractSection/index.tsx [hunk 2]
```diff
@@ -344,3 +355,62 @@ export const StakingContractSection = ({
...
+
+  const minutes = Math.floor(totalSeconds / 60);
+  const seconds = totalSeconds % 60;
+
+  // Ensure double digits for hours, minutes, and seconds
+  const formattedHours = String(hours).padStart(2, '0');
+  const formattedMinutes = String(minutes).padStart(2, '0');
+  const formattedSeconds = String(seconds).padStart(2, '0');
+
+  return `${days} days ${formattedHours} hours ${formattedMinutes} minutes ${formattedSeconds} seconds`;
```
[the remark this answers is not in the document]
Comment (truemiller) [reply frontend/components/ManageStakingPage/StakingContractSection/index.tsx:415]: thanks, had overlooked this!
should be good on decimals, we use `Math.floor` on ln 407

**Review by solarw — APPROVED** [review]
lgtm

**Comment by truemiller** [comment]
@mohandast52 I'll open a follow-up for the <Text> component change and plural conditions

**Pull request merged**